### ISSUE TITLE
"Back" button reads as "Select"

### ISSUE THREAD

**Issue by biomagnet** [Vita3K/Vita3K-Android#601]
On the [Retroid Pocket 4 Pro](https://www.goretroid.com/collections/frontpage/products/retroid-pocket-4-handheld), the physical "Back" button only registers as the same input as "Select". On other emulators such as NetherSX2 and Citra (rip in peace), the back button works as intended to bring up the emulation menu, same as using the swipe method to hit back. In Vita3k, it assigns key "4" no matter if I hit Select or Back.

Since hitting "Back" in-game does nothing, I'd like to assign that button as the PS Button in settings but obviously this puts a damper on it.

**Comment by windwaker87** [comment]
The same on `AYN Odin 2`. Need to use on-screen navigation controls to get back to the vita3k home screen.

However, the hardware `back` button on `Anbernic RG505` works correctly right out of the box.

**Comment by leira** [comment]
Got the exact same issue on Metroid pocket flip 2.

**Comment by biomagnet** [comment]
Yep, even now that I have a Retroid Pocket 5, the same issue remains. I assume it's some combination of how Retroid has their XBox drivers set up, but if that's the case, an exception of some sort might be feasible? Especially if other emulators can detect the different buttons properly. I don't know.

**Comment by yawnone-max** [comment]
same problem here on my Anbernic Rg476h, how do I get back to the main menu?

**Comment by manzaloros** [comment]
Same issue on the RG 477m.

**Comment by kimujifour** [comment]
Same issue on AYN Odin 3.

**Comment by Draggoman-1** [comment]
Same on AYN THOR
